### DISCUSSION TITLE
How to display the contents of my ASP.NET custom module created inside orchard CMS?

### DISCUSSION THREAD

**Discussion by PaariLaishram** [OrchardCMS/OrchardCore#16645] [Q&A]
Hi, I am trying to integrate my ASP.NET project with orchard. I have got an ASP.NET custom module created inside my Orchard CMS by following the steps provided over [here](https://docs.orchardcore.net/projects/O1/en/latest/Documentation/Building-a-hello-world-module/). I can see the contents of this module when I go to the custom module route (/helloWorld). But how do I convert this custom module to an orchard content like the default 'Welcome to Orchard' content with view and editing features.

I tried creating a content from Ochard CMS > Content > Create New Content but i figured out this was actually creating a new content through orchard and not taking my custom module code

**Comment by hishamco** [comment]
The link you are referring to is Orchard 1, try to follow Orchard Core docs https://docs.orchardcore.net/

- Simply you can create an Orchard Core module by using our custom project templates
- Reference the project in the application host (Web App)
- Go to the admin site and enable your module
